ui/packages/ui/src/app/pages/createConnector/CreateConnectorPage.tsx

```diff
@@ -307,11 +309,31 @@ export const CreateConnectorPage: React.FunctionComponent = () => {
     validateOptionProperties(propertyValues, propertyCategory);
   };
 
+  const validateConnectionName = (connName: string): string => {
+    const currentNames = location.state?.connectorNames;
```

Comment:
Hey, @mdrillin I was thinking if we need to add ignore case to name. Anyway, I have asked the same to gunner and Rene on the GitHub issue.